cmd/xl-v1-object.go

```diff
@@ -340,7 +341,7 @@ func (xl xlObjects) getObjectInfoDir(ctx context.Context, bucket, object string)
 		}, index)
 	}
 
-	readQuorum := len(storageDisks) / 2
+	readQuorum := int(math.Ceil(float64(len(storageDisks)) / 2.0))
```

Comment:
```suggestion
	readQuorum := readQuorum(len(storageDisks))
```
... or just inline the call below.